Geospatial/Travel Times/travel times functions.R

```diff
@@ -0,0 +1,68 @@
+
+# get_travel_areas - Function to obtain the area that is travellable from location (longitude, latitude) within
+#                    a specified travel_time using a particular travel_method.
+# - longitude     - vector of longitudes of locations being travelled from
+# - latitude      - vector of latitudes of locations being travelled from
+# - travel_time   - Travel time of interest in minutes. Recommend that this be no less than 10
+# - travel_method - Method of travel. One of "car", "bike", or "foot"
+# - within_region - Optional region that the travel area should remain within. object of class sf
+
+get_travel_areas <- function(longitude, latitude, travel_time, travel_method = "car", within_region = NULL){
+  # Get travel areas
+  iso <- purrr::map2(longitude, latitude, # Iterate over each point (slow)
+    \(x, y) get_iso(x, y, travel_time, travel_method)
+  )
+  
+  # remove empty areas - generally happens when travel time is set < 10
+  iso <- iso[sapply(iso, nrow) != 0]
+  
+  # Combine into a single area to make plotting easier  
+  iso <- suppressWarnings(purrr::reduce(iso, \(x, y) st_make_valid(x) %>% st_union(y)))
```

Comment:
If `suppressWarnings` is absolutely necessary can you add a comment as to why?

Reply:
Done - happy for you to judge if reasoning for use is sound. Removed later in code as this is merging two different objects.